x11/tktable/files/patch-INT2PTR.diff

```diff
@@ -0,0 +1,798 @@
++$ac_includes_default
++int
++main ()
++{
++if (sizeof ((uintptr_t)))
++	  return 0;
++  ;
++  return 0;
++}
++_ACEOF
++rm -f conftest.$ac_objext
++if { (ac_try="$ac_compile"
++case "(($ac_try" in
++  *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
++  *) ac_try_echo=$ac_try;;
++esac
++eval ac_try_echo="\"\$as_me:$LINENO: $ac_try_echo\""
++$as_echo "$ac_try_echo") >&5
++  (eval "$ac_compile") 2>conftest.er1
++  ac_status=$?
++  grep -v '^ *+' conftest.er1 >conftest.err
++  rm -f conftest.er1
++  cat conftest.err >&5
++  $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
++  (exit $ac_status); } && {
++	 test -z "$ac_c_werror_flag" ||
++	 test ! -s conftest.err
++       } && test -s conftest.$ac_objext; then
++  :
++else
++  $as_echo "$as_me: failed program was:" >&5
++sed 's/^/| /' conftest.$ac_ext >&5
++
++	ac_cv_type_uintptr_t=yes
++fi
++
++rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
++else
++  $as_echo "$as_me: failed program was:" >&5
++sed 's/^/| /' conftest.$ac_ext >&5
++
++
++fi
++
++rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
++fi
++{ $as_echo "$as_me:$LINENO: result: $ac_cv_type_uintptr_t" >&5
++$as_echo "$ac_cv_type_uintptr_t" >&6; }
++if test "x$ac_cv_type_uintptr_t" = x""yes; then
++
++cat >>confdefs.h <<\_ACEOF
++#define HAVE_UINTPTR_T 1
++_ACEOF
++
++else
++  for ac_type in 'unsigned int' 'unsigned long int' \
++	'unsigned long long int'; do
++       cat >conftest.$ac_ext <<_ACEOF
++/* confdefs.h.  */
++_ACEOF
++cat confdefs.h >>conftest.$ac_ext
++cat >>conftest.$ac_ext <<_ACEOF
++/* end confdefs.h.  */
++$ac_includes_default
++int
++main ()
++{
++static int test_array [1 - 2 * !(sizeof (void *) <= sizeof ($ac_type))];
++test_array [0] = 0
++
++  ;
++  return 0;
++}
++_ACEOF
++rm -f conftest.$ac_objext
++if { (ac_try="$ac_compile"
++case "(($ac_try" in
++  *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
++  *) ac_try_echo=$ac_try;;
++esac
++eval ac_try_echo="\"\$as_me:$LINENO: $ac_try_echo\""
++$as_echo "$ac_try_echo") >&5
++  (eval "$ac_compile") 2>conftest.er1
++  ac_status=$?
++  grep -v '^ *+' conftest.er1 >conftest.err
++  rm -f conftest.er1
++  cat conftest.err >&5
++  $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
++  (exit $ac_status); } && {
++	 test -z "$ac_c_werror_flag" ||
++	 test ! -s conftest.err
++       } && test -s conftest.$ac_objext; then
++
++cat >>confdefs.h <<_ACEOF
++#define uintptr_t $ac_type
++_ACEOF
++
++	  ac_type=
++else
++  $as_echo "$as_me: failed program was:" >&5
++sed 's/^/| /' conftest.$ac_ext >&5
++
++
++fi
++
++rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
++       test -z "$ac_type" && break
++     done
++fi
++
+ 
+ 
+ #--------------------------------------------------------------------
+ # Everyone should be linking against the Tcl stub library.  If you
+ # can't for some reason, remove this definition.  If you aren't using
+
+Index: configure.in
+==================================================================
+--- configure.in
++++ configure.in
```

Comment:
A patch that modifies both configure and configure.in, in that order, can cause autoconf to be run unexpectedly (and that could fail) if the clock second when configure gets modified is not the same as the clock second when configure.in gets modified. The solution is to either patch only configure, or patch only configure.in and run autoreconf, or patch configure.in before patching configure (list configure.in before configure in the patchfile).

Reply:
Thank you for pointing this out. I have opted for patching configure before configure.in.

Reply:
configure.in before configure, sorry